bot/seasons/evergreen/magic_8ball.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Magic8ball:
+    """
+    A Magic 8ball command to respond to a users question.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        with open(Path("bot", "resources", "evergreen", "magic8ball.json"), "r") as file:
+            self.answers = json.load(file)
+
+    @commands.command(name="8ball")
+    async def output_answer(self, ctx, question: list):
+        """
+        Return a magic 8 ball answer from answers list.
+        """
+        if len(question) >= 3:
```

Comment:
The user should get some feedback if this criteria isn't met rather than failing silently.

Reply:
I was under the impression that this was giving the user some feedback. The bout outputs a message when the criteria is not met.
![image](https://user-images.githubusercontent.com/1500344/54882296-d6c1d480-4e2e-11e9-9918-266d6d8e73e4.png)

Reply:
Are you suggesting, something like a command usage output on fail instead?

Reply:
That is only shown by the [error handler](https://github.com/python-discord/seasonalbot/blob/master/bot/seasons/evergreen/error_handler.py) if `commands.UserInputError` is raised, which doesn't happen if `question` is shorter than 3 characters (nothing happens).